docs/en/02_site-wide-content-report.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Site-Wide Content Report
+summary: Customizing the columns of the site-wide Content Report and improvoing performance of large datasets
+icon: file-alt
+---
+
+# Site-wide content report
+
+## Customising the report
+
+In order to customise the columns included in a report you can build a custom extension and apply it to the
+SitewideContentReport as necessary.
+
+The built in [`SitewideContentTaxonomy`](api:SilverStripe\Reports\SiteWideContentReport\Extensions\SitewideContentTaxonomy) extension, for instance, adds a custom columns extracted from the `silverstripe/taxonomy` modules, and can be used as a base for developing further extensions:
```

Comment:
```suggestion
The built in [`SitewideContentTaxonomy`](api:SilverStripe\Reports\SiteWideContentReport\Extensions\SitewideContentTaxonomy) extension, for instance, adds a custom columns extracted from the `silverstripe/taxonomy` module, and can be used as a base for developing further extensions:
```
Please also link to the taxonomy docs in the optional features section

Reply:
Updated